main.go

```
@@ -239,6 +240,7 @@ func initLedger(ctx *cli.Context) (*ledger.Ledger, error) {
 
 	var err error
 	dbDir := utils.GetStoreDirPath(config.DefConfig.Common.DataDir, config.DefConfig.P2PNode.NetworkName)
+	ledgerstore.CheckStorage(dbDir)
```

why ignore error handle?